### ISSUE TITLE
wiwo.de: brightcove.py xml.etree.ElementTree.ParseError: mismatched tag

### ISSUE THREAD

**Issue by probonopd** [rg3/youtube-dl#4849]
The following error occurs when trying to use youtube-dl with wiwo.de:

```
youtube-dl  http://www.wiwo.de/videos/boersen-news/ezb-geldflut-stuetzt-kurse-zu-wochenbeginn/11317308.html
[generic] 11317308: Requesting header
WARNING: Falling back on generic information extractor.
[generic] 11317308: Downloading webpage
[generic] 11317308: Extracting information
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl", line 8, in <module>
    load_entry_point('youtube-dl==2015.02.02', 'console_scripts', 'youtube-dl')()
  File "/Library/Python/2.7/site-packages/youtube_dl/__init__.py", line 382, in main
    _real_main(argv)
  File "/Library/Python/2.7/site-packages/youtube_dl/__init__.py", line 372, in _real_main
    retcode = ydl.download(all_urls)
  File "/Library/Python/2.7/site-packages/youtube_dl/YoutubeDL.py", line 1350, in download
    res = self.extract_info(url)
  File "/Library/Python/2.7/site-packages/youtube_dl/YoutubeDL.py", line 619, in extract_info
    ie_result = ie.extract(url)
  File "/Library/Python/2.7/site-packages/youtube_dl/extractor/common.py", line 267, in extract
    return self._real_extract(url)
  File "/Library/Python/2.7/site-packages/youtube_dl/extractor/generic.py", line 743, in _real_extract
    bc_urls = BrightcoveIE._extract_brightcove_urls(webpage)
  File "/Library/Python/2.7/site-packages/youtube_dl/extractor/brightcove.py", line 187, in _extract_brightcove_urls
    return [cls._build_brighcove_url(m) for m in matches]
  File "/Library/Python/2.7/site-packages/youtube_dl/extractor/brightcove.py", line 119, in _build_brighcove_url
    object_doc = xml.etree.ElementTree.fromstring(object_str.encode('utf-8'))
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xml/etree/ElementTree.py", line 1300, in XML
    parser.feed(text)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xml/etree/ElementTree.py", line 1642, in feed
    self._raiseerror(v)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xml/etree/ElementTree.py", line 1506, in _raiseerror
    raise err
xml.etree.ElementTree.ParseError: mismatched tag: line 13, column 2
```

**Comment by phihag** [comment]
Thank you for the report. I believe this issue to be fixed in youtube-dl 2015.02.02.5 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.

**Comment by probonopd** [comment]
Works for me, thank you very much.
